cmd/sourced/cmd/status.go

```diff
@@ -2,18 +2,165 @@ package cmd
 
 import (
 	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
 
 	"github.com/src-d/sourced-ce/cmd/sourced/compose"
+	"github.com/src-d/sourced-ce/cmd/sourced/compose/workdir"
 )
 
 type statusCmd struct {
-	Command `name:"status" short-description:"Show the status of all components" long-description:"Show the status of all components"`
+	Command `name:"status" short-description:"Show the list of working directories and the current deployment" long-description:"Show the list of working directories and the current deployment"`
 }
 
-func (c *statusCmd) Execute(args []string) error {
+type statusAllCmd struct {
+	Command `name:"all" short-description:"Show all the available status information" long-description:"Show all the available status information"`
+}
+
+func (c *statusAllCmd) Execute(args []string) error {
+	fmt.Print("List of all working directories:\n")
+
+	err := printWorkdirsCmd()
+	if err != nil {
+		printRed(err.Error())
+	}
+
+	active, err := activeWorkdir()
+	if isNotExist(err) {
+		// skip printing the config and components when there is no active dir
+		return nil
+	}
+
+	if err != nil {
+		return err
```

Comment:
why you return an error here but in other places just `printRed`? Maybe we should make it consistent?